GliaWidgetsTests/SecureConversations/Welcome/SecureConversations.WelcomeViewModelSpec.swift

```diff
@@ -688,10 +697,11 @@ extension SecureConversationsWelcomeViewModelTests {
         )
 
         XCTAssertEqual(viewModel.availabilityStatus, .unavailable(.unauthenticated))
-        XCTAssertEqual(
-            alertConfiguration?.message,
-            viewModel.environment.alertConfiguration.unavailableMessageCenterForBeingUnauthenticated.message
-        )
+        if case .unavailableMessageCenterForBeingUnauthenticated = alertInputType {
```

Comment:
I think this can be simplified by introducing `Bool` constant before `if/else` and assign value to it In `if/else` branches, then `XCTAssertTrue` it once.

Reply:
fixed [c08f255](https://github.com/salemove/ios-sdk-widgets/pull/968/commits/c08f255d539b691b212f88c3695b2f272a154aad)